pkgs/development/python-modules/ml-dtypes/default.nix

```diff
@@ -31,6 +31,7 @@ buildPythonPackage rec {
       --replace "numpy~=1.21.2" "numpy" \
       --replace "numpy~=1.23.3" "numpy" \
       --replace "numpy~=1.26.0" "numpy" \
+      --replace "numpy==2.0.0rc1" "numpy" \
```

Comment:
That's a funny list. More reliable than a regex I guess xD

Reply:
X)

Reply:
`pythonRelaxDeps`?

Reply:
Doesn't work here unfortunately...

Reply:
```
ERROR Missing dependencies:
        numpy==2.0.0rc1
        setuptools~=68.1.0

Reply:
this seems like a `pythonRelaxDepsHook` bug, no?